src/wallet/interfaces.cpp

```diff
@@ -530,7 +530,7 @@ class WalletClientImpl : public WalletClient
     //! WalletClient methods
     std::unique_ptr<Wallet> createWallet(const std::string& name, const SecureString& passphrase, uint64_t wallet_creation_flags, bilingual_str& error, std::vector<bilingual_str>& warnings) override
     {
-        std::shared_ptr<CWallet> wallet;
+        std::shared_ptr<CWallet> const wallet;
```

Comment:
Declaring a local `const std::shared_ptr<T> const_ptr;` is pointless. It can't be set to anything after this declaration since it's const.

In fact, this `wallet` variable isn't used, so I think it can just be removed.

Reply:
Yes, this one just sorta tagged along. I have no idea why it is there, so I didn't touch it beyond letting it tag along with the convention change.